etcdmain/etcd.go

```diff
@@ -56,7 +56,16 @@ func startEtcdOrProxyV2() {
 	defaultInitialCluster := cfg.ec.InitialCluster
 
 	err := cfg.parse(os.Args[1:])
+	var zapError error
```

Comment:
can we move `zapError` below, since it's not used after `if lg == nil` block

Reply:
@gyuho sure, sounds good. Thanks!